src/schemas/dataset.schema.json

```diff
@@ -0,0 +1,59 @@
+{
+  "type": "object",
+  "description": "Dataset",
+  "title": "IDataset",
+  "required": ["_type", "components", "label", "primary_owner"],
+  "additionalProperties": false,
+  "properties": {
+    "_type": {
+      "const": "glue.core.data.Data"
+    },
+    "components": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "coords": {
+      "type": "string"
+    },
+    "label": {
+      "type": "string"
+    },
+    "meta": {
+      "type": "object",
+      "$ref": "#/definitions/meta"
+    },
+    "primary_owner": {
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "style": {
+      "type": "object",
+      "$ref": "#/definitions/style"
+    }
+  },
+  "definitions": {
+    "meta": {
+      "type": "object",
+      "properties": {
+        "_type": {
```

Comment:
the schema of viewers needs to have this `_type` property because it is defined in the glue session file. If this data does not preset in the dataset section of the glue file, I think we don't need it in the schema.